crates/starknet-server/tests/test_dump_and_load.rs

```diff
@@ -332,4 +346,116 @@ mod dump_and_load_tests {
             panic!("Could not unpack the transaction from {loaded_transaction:?}");
         }
     }
+
+    #[tokio::test]
+    async fn dump_load_dump_load_on_exit() {
+        let dump_file = UniqueAutoDeletableFile::new("dump_load_dump_load_on_exit");
+        let devnet_dump_first = BackgroundDevnet::spawn_with_additional_args(&[
```

Comment:
First two devnets can be handled in a loop (to reduce test size)

Reply:
I'll refactor `dump_load_dump_load_on_exit` and `dump_load_dump_load_on_transaction` tests to reduce code size

Reply:
Fixed in a29d86e83832f7d8d310bb0791efd10472a27666

Reply:
I attached this issue to this PR